### DISCUSSION TITLE
1:40:00 - 1:45:00 (Random tensors)

### DISCUSSION THREAD

**Discussion by 123321prog** [mrdbourke/pytorch-deep-learning#964] [Q&A]
when i create a random tensor, it always said the random_tensor.ndim was 2

**Comment by OneManShow0815** [comment]
Can you show a screenshot?
ndim is the number of dimensions of a tensor. If you are using a=torch.randn(3,2) you have 2 dimensions. If you use a=torch.randn(6,8) its still 2 dimensions. a=torch.randn(3,2,4) now has 3 dimensions.
Please try and send a screenshot.
